src/Select/SelectInput.js

```diff
@@ -114,14 +114,15 @@ class SelectInput extends React.Component<AllProps, State> {
     });
   };
 
-  handleItemClick = (child: Element<*>) => (event: SyntheticMouseEvent<>) => {
+  handleItemClick = (child: Element<*>) => (event: SyntheticMouseEvent<> & { target: Object }) => {
```

Comment:
No, you are artificially claiming that anything passed into here has a `target`, which is not true.

Reply:
If I understand properly, `target` should be optional here, and also before doing `...event.target` I should check if `target` is present in `event`. Am I right?

Reply:
In the above, you are fabricating a new type using an intersection.  Since we are not in control of these types, we should not be trying to alter them.

Checking presence of target is the right thing to do, since we cannot have concrete type information at this point in the handling process.